### DISCUSSION TITLE
apple test device minimum requirements

### DISCUSSION THREAD

**Discussion by sjoerd75** [dotnet/maui#17888] [Q&A]
Hello!

 To be able to test mac and iphone platforms, could I use a mac from (2009)? 

With kind regards, 
Sjoerd

**Comment by AndreasReitberger** [comment]
Propably not, I had a 2014 MacBook and due to the support end of the latest updates (XCode and the MacOS version itself), I needed to replace the MacBook with a newer one.
So either you can upgrade your MacBook to the latest MacOS and XCode version, or you need to get a newer one.
At least for building and running your apps on the MacBook while debugging.

**Reply by sjoerd75** [reply]
Thank you for your clear and quick reply! Another question then, since I plan to develop on a pc, but to test the Mac version of the application, could I use such older models to run the mac and iphone app emulator?

**Reply by AndreasReitberger** [reply]
You need the latest XCode version installed on the MacBook. If this is possible on your device, you can run and debug your apps on the Mac and the simulators.

You just check this by connecting with your computer to your mac with visual studio.
If your Xcode version is not supported anymore, you'll get a message then. This is the easiest way to verify this.
